.nuget/create_nuget.py

```diff
@@ -45,8 +45,8 @@
         <file src="runtimes\**" target="runtimes"/>
         </files>
     </package>
-    """ % (version, datetime.datetime.now().year)
-    prop_str = r"""
+    """ 
+    prop_str = f"""
```

Comment:
```suggestion
    prop_str = r"""
```

This needs to be a raw string and does not rely on any Python template. Please change this back.

Reply:
I have added the requested changes to create_nuget.py.